runtime/sema/checker.go

```diff
@@ -1656,6 +1656,8 @@ func (checker *Checker) ResetErrors() {
 	checker.errors = nil
 }
 
+const invalidTypeDeclarationAccessModifierExplanation = "type declarations must be public for now"
```

Comment:
```suggestion
const invalidTypeDeclarationAccessModifierExplanation = "type declarations must be public"
```

I feel like it doesn't really need the `for now` part.